src/mixins/filing-template-mixin.ts

```diff
@@ -45,27 +48,36 @@ export default class FilingTemplateMixin extends Vue {
       filing: {
         header: {
           name: INCORPORATION_APPLICATION,
-          certifiedBy: this.stateModel.certifyState.certifiedBy,
-          email: this.stateModel.defineCompanyStep.businessContact.email,
-          date: this.stateModel.currentDate
+          certifiedBy: this.stateModel?.certifyState?.certifiedBy ?? '',
```

Comment:
Why use ?? instead of || ?

Reply:
Checking for null or undefined...falsy values are cool here...but I suppose I could use || as well...do we have a strong preference?

Reply:
also...I'm a C# dev historically :p

Reply:
Technically those are OK here but there's little usage of ?? so far and it comes with a gotcha (doesn't work for some falsy values) so my preference is ||.

Reply:
I think for consistency, we should stick with `||` as well. 
I couldn't tell you if one is better than the other and if there is a good reason to use `??` over `||` then i'm all for the change, but if its both relatively the same, i'd try to be as congruent as possible.

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator

Reply:
changed

Reply:
oh nice, thanks for the reference. That is a pretty good explanation!